nixos/modules/system/boot/initrd-openvpn.nix

```diff
@@ -71,6 +78,15 @@ in
     boot.initrd.network.postCommands = ''
       openvpn /etc/initrd.ovpn &
     '';
+
+    boot.initrd.systemd.services.openvpn = {
+      wantedBy = ["initrd.target"];
+      path = [pkgs.iproute2];
+      after = ["network.target" "initrd-nixos-copy-secrets.service"];
+      serviceConfig.ExecStartPre = "/bin/mkdir /tmp";
```

Comment:
This will need to be changed once staging reaches master because of https://github.com/NixOS/nixpkgs/pull/216826/files#diff-68fef76b66b99ee77baba62969a434c1f912c2fcf8765bfc884b7aa5b0e70d35R375